packages/experiments/src/components/Tile/examples/Tile.Media.Example.tsx

```diff
@@ -13,22 +13,82 @@ import {
 import { lorem } from '@uifabric/example-app-base';
 import * as TileExampleStylesModule from './Tile.Example.scss';
 
-const ITEMS: { name: string; activity: string }[] = [
+const ITEMS: { name: JSX.Element; activity: JSX.Element }[] = [
   {
-    name: lorem(2),
-    activity: lorem(6)
+    name: <>{lorem(2)}</>,
+    activity: (
+      <>
+        <span>267 &times; 200&ensp;&middot;&ensp;123.45&nbsp;MB</span>
+        <br />
+        <SignalField
+          before={[
+            <Signal key={0}>
+              <Icon iconName="play" />
+            </Signal>,
+            <MentionSignal key={1} />
+          ]}
+        >
+          {lorem(6)}
+        </SignalField>
+      </>
+    )
   },
   {
-    name: lorem(2),
-    activity: lorem(6)
+    name: <>{lorem(2)}</>,
+    activity: (
+      <>
+        <span>200 &times; 267&ensp;&middot;&ensp;123.45&nbsp;MB</span>
+        <br />
+        <SignalField
+          before={[
+            <Signal key={0}>
+              <Icon iconName="play" />
+            </Signal>,
+            <SharedSignal key={1} />
+          ]}
+        >
+          {lorem(6)}
+        </SignalField>
+      </>
+    )
   },
   {
-    name: lorem(2),
-    activity: lorem(6)
+    name: <>{lorem(2)}</>,
+    activity: (
+      <div>
+        <span>200 &times; 200&ensp;&middot;&ensp;123.45&nbsp;MB</span>
+        <br />
+        <SignalField
+          before={[
+            <Signal key={0}>
+              <Icon iconName="play" />
+            </Signal>,
+            <MentionSignal key={1} />
+          ]}
+        >
+          {lorem(6)}
+        </SignalField>
+      </div>
+    )
   },
   {
-    name: lorem(2),
-    activity: lorem(6)
+    name: <>{lorem(2)}</>,
+    activity: (
+      <div>
+        <span>180 &times; 180&ensp;&middot;&ensp;123.45&nbsp;MB</span>
+        <br />
```

Comment:
We should avoid using `<br />` in favor of just having elements properly use `display: block` to enforce newlines.